package.json

```diff
@@ -80,51 +80,61 @@
       {
         "command": "cmake.viewLog",
         "title": "%cmake-tools.command.cmake.viewLog.title%",
+        "when": "!cmake:cmtPartiallyActive",
```

Comment:
I don't think these changes belong in this PR

Reply:
ups